src/get-variables.js

```diff
@@ -1,7 +1,7 @@
 import stripComments from 'strip-json-comments';
 
 export default function getVariables(content) {
-  const variableRegex = /\$(.+):\s+(.+);/;
+  const variableRegex = /\$(.+):\s+(.+);?/;
```

Comment:
Well no, because we are doing a split on `\n` before this is used. If we wanna be really strict we can check for `$` (end of string) but I don't see the use since we throw everything out of the  two groups, `()`, away anyway.